core/capabilities.go

```diff
@@ -115,6 +116,7 @@ var CapabilityNameLookup = map[Capability]string{
 	Capability_ImageToVideo:               "Image to video",
 	Capability_Upscale:                    "Upscale",
 	Capability_AudioToText:                "Audio to text",
+	Capability_LLM:                        "Large Language Model",
```

Comment:
This needs to update to `Large language model` to work with the `PipelineToCapability` function.  I know this is brittle, will look at making it better going forward.

Reply:
There's no reason to have this mapping part of the core package.

It's only used for monitoring and should live in the monitoring package if we want it for readability of monitoring. 

There's also inconsistent usage of a helper and direct mapping looking.

I really don't see much utility for this mapping other than as you say, a brittle part of the codebase.

Reply:
Thanks for applying the commit. We can remove it or improve it in the future 👍🏻.